unit-tests/unit-test-config.py

```diff
@@ -14,23 +14,60 @@
 # process and so individual tests cannot affect others except through hardware.
 #
 
-import sys, os, subprocess, locale, re
+import sys, os, subprocess, locale, re, getopt
 from glob import glob
 
-if len(sys.argv) != 3:
+current_dir = os.path.dirname( os.path.abspath( __file__ ) )
+sys.path.append( current_dir + os.sep + "py" )
+
+from rspy import file, repo, unittest
```

Comment:
How about `log`?

Reply:
Wanted to ask you about that, If you want all the prints to become logs and get the `-E-` and all